backend/curation/api/curation-api.yml

```diff
@@ -1153,6 +1171,10 @@ components:
           $ref: "#/components/schemas/dataset_published_at"
         revised_at:
           $ref: "#/components/schemas/dataset_revised_at"
+        revision_of_collection:
+          $ref: "#/components/schemas/collection_revision_of"
+        revision_of_dataset:
+          $ref: "#/components/schemas/dataset_revision_of"
```

Comment:
change this, and then the place where it is defined, to `revision_of_dataset`

Reply:
I am following the pattern of [collection_revision_of](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/backend/curation/api/curation-api.yml#L851) here; are you OK breaking with consistency here?